src/widgets/modals/RegimenDataModal.js

```diff
@@ -71,13 +71,13 @@ export const RegimenDataModal = ({ requisition }) => {
     });
   }, [customData]);
 
-  const getAction = columnKey => {
+  const getCallback = columnKey => {
     switch (columnKey) {
       default:
       case 'comment':
-        return actions.updateComment;
+        return (value, rowKey) => dispatch(actions.updateComment(value, rowKey));
       case 'value':
-        return actions.updateValue;
+        return (value, rowKey) => dispatch(actions.updateValue(value, rowKey));
```

Comment:
Just for consistency, maybe give these identifiers?

Reply:
276bade